src/libraries/Microsoft.XmlSerializer.Generator/src/Microsoft.XmlSerializer.Generator.csproj

```diff
@@ -15,6 +15,13 @@
     <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
   </PropertyGroup>
 
+  <!-- Disable binplacing -->
+  <PropertyGroup>
+    <BinPlaceRef>false</BinPlaceRef>
+    <BinPlaceRuntime>false</BinPlaceRuntime>
+    <BinPlaceForTargetVertical>false</BinPlaceForTargetVertical>
+  </PropertyGroup>
+
```

Comment:
We hit issues in the past with this assembly unintentionally being picked up as part of the shared framework etc, I think you suggested back then to disable binplacing (since it's not necessary for this assembly).

Reply:
@ViktorHofer updated to `EnableBinPlacing`

Reply:
How is this different than any other OOB assembly like Microsoft.Extensions.* ones which we binplace as well and which aren't part of the shared framework. Should we stop binplacing them as well in the future?